### PR TITLE
Updated front page

### PR DESCRIPTION
**Pull request by leahcimto**
There is a new application system, so I updated index.html to reflect that. I also added a link for the studio instead of it saying “(there is a link in the header)”.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,9 +19,9 @@ <h2>Who are we?</h2>
...
 			<h3>I submitted my application! Now what?</h3>
-			<p style="margin-left:200px;margin-right:200px">Once you submit an application, three of the Studio Managers will look at your application and say if they think you would be a good fit for this group. If all three managers agree that you would make a good fit, you will be invited to the studio. If they don't all agree, you can reapply in a week! There is currently a limit of 6 pending applications at a time.</p>
+			<p style="margin-left:200px;margin-right:200px">Once you submit an application, a few of the studio curators and managers will look at your application and say if they think you would be a good fit for this group. If all the curators and managers think that you would make a good fit, you will be invited to the studio. If they don't all agree, you can reapply in a week! You will get a comment telling you if you were accepted, as well as an invite to the studio, and if you were rejected you will get a message saying how you can improve. There is currently a limit of 6 pending applications at a time.</p>
```
Comment (Accio1) [inline index.html:24]: Please change this to: 
"Once you submit an application, a few of the studio curators and managers will look at your application and say if they think you would be a good fit for this group. If several curators and managers think that you would make a good fit, your application will be accepted and you will be informed and invited to the studio. If they don't think you are a good fit at this time, or if a week passes without you being accepted, then your application will be rejected and you will get a message saying how you can improve. If your application has been rejected, feel free to re apply in two weeks, keeping in mind the feedback that you were given. While your application is being discussed, feel free to look at the discussion, but please don't comment on it, or you will lose a point and your chances of being accepted will be lower. Feel free to withdraw your application at any time, but keep in mind that you will have to wait 2 weeks before re applying if you withdraw your application. There is currently a limit of 6 pending applications at a time."

**Review by Accio1 — CHANGES_REQUESTED** [review]
A few wording changes and then this will be good to merge!

**Comment by leahcimto** [comment]
Thanks, I’ll do that when I have time.

**Comment by leahcimto** [comment]
@Accio1 Done, let me know if anything else should be changed.

**Review by Accio1 — APPROVED** [review]
Thank you! Merging!

**Pull request merged**